docs/reference/redirects.asciidoc

```diff
@@ -158,10 +158,21 @@ See <<grok,Grokking grok>>.
 
 // [START] Security redirects
 
+[role="exclude",id="security-minimal-setup"]
+=== Set up minimal security for {es}
+
+Refer to <<manually-configure-security,Manually configure security for the {stack}>>.
+
 [role="exclude",id="get-started-users"]
-=== Create users
+==== Create users
+
+Refer to the <<reset-password,`elasticsearch-reset-password`>> tool for resetting
+passwords of built-in users.
```

Comment:
Are we documenting resetting the (built-in) users passwords anywhere?
It can be done (for all users) with the `elasticsearch-reset-password` tool or the API directly (I lost track which approach was favored).

Reply:
We could consider adding a page for changing passwords for native and built-in users and indicate that users can use the `elasticsearch-reset-password` tool or the change passwords API. The former is easier to use and auto-generates a password, where the latter allows users to specify a specific password.

Reply:
You can specify a password when using the tool too, see the `-i` flag